src/bundle/Resources/public/img/ez-icons.svg

```diff
@@ -638,4 +638,10 @@
 <symbol id="workflow" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
     <path d="M8.667 15.667v1c0 .736-.597 1.333-1.333 1.333h-3a1.335 1.335 0 0 1-.143-2.666h.003c-1.527-1.317-2.193-3.107-2.193-5.667 0-3.203 1.643-5.69 4.75-7.2a1.334 1.334 0 0 1 1.174 2.397l-.008.003c-2.187 1.06-3.25 2.63-3.25 4.8 0 1.817.35 2.913 1.36 3.747a1.333 1.333 0 0 1 2.64.253v2.001zM27.807 27C29.334 25.683 30 23.893 30 21.333c0-3.203-1.643-5.69-4.75-7.2a1.334 1.334 0 0 0-1.174 2.397l.008.003c2.187 1.06 3.25 2.63 3.25 4.8 0 1.817-.333 2.913-1.36 3.747a1.333 1.333 0 0 0-2.64.253v3.001c0 .736.597 1.333 1.333 1.333h3l.07.002a1.335 1.335 0 0 0 .073-2.668h-.003zM16 22.667a4.667 4.667 0 1 0 4.667 4.667A4.667 4.667 0 0 0 16 22.667zm4.667-4v-5.333a2 2 0 0 0-2-2h-5.333a2 2 0 0 0-2 2v5.333a2 2 0 0 0 2 2h5.333a2 2 0 0 0 2-2zM14 14h4v4h-4zm6.667-9.333A4.667 4.667 0 1 0 16 9.334a4.667 4.667 0 0 0 4.667-4.667zm-2.667 0a2 2 0 1 1-2-2 2 2 0 0 1 2 2z"></path>
 </symbol>
+<symbol id="approved" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+    <path d="M16.177 22c-.357 0-.681-.141-.92-.37L7.08 13.817a1.333 1.333 0 1 1 1.84-1.926l7.23 6.907L29.717 5.065a1.334 1.334 0 1 1 1.9 1.874l-14.49 14.667a1.33 1.33 0 0 1-.949.397h-.001zM32 16c0-1.289-.151-2.542-.435-3.743l.022.11a1.333 1.333 0 1 0-2.585.61l-.001-.007c.224.915.353 1.966.353 3.047 0 7.364-5.97 13.333-13.333 13.333S2.688 23.38 2.688 16.017c0-7.364 5.97-13.333 13.333-13.333a13.28 13.28 0 0 1 8.493 3.054l-.022-.018a1.333 1.333 0 0 0 1.702-2.051l-.003-.002A15.907 15.907 0 0 0 16.001.002c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16V16z"></path>
+</symbol>
+<symbol id="rejected" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would rename it to `workflow-rejected`

Reply:
It was named `rejected` on purpose because AdminUI should not have dependency on Workflow at all. It's weird to me that we store icons used by other packages in AdminUI repository.

Naming it `rejected` and `approved` makes them general purpose icons.

Reply:
Yes, but if you look closer to the icon itself, then you'll notice that it says nothing about being rejected. It's more like revert